utopia-vscode-extension/snippets.json

```diff
@@ -0,0 +1,15 @@
+{
+  "registerComponent": {
+    "prefix": "registerComponent",
+    "body": [
+      "registerComponent({",
+      "  name: ${1}, // name is what appears in the insertion list",
+      "  moduleName: ${2}, // name of module, or relative file path",
+      "  controls: ${3}, // property controls",
```

Comment:
I think it's worth also including a snippet for a very basic control here. Perhaps a dummy string-input control would be enough. I say this because I don't think there's any way we could otherwise provide a code snippet for the property controls (unless you can think of one?), and without that it's still a bit too unclear how to proceed.

Reply:
I added a dummy control here. 
In general the property controls need more snippets and examples about how to write them.